app/controllers/appointments_controller.rb

```diff
@@ -0,0 +1,4 @@
+class AppointmentsController < ApplicationController
```

Comment:
We need a before_action for this controller so that logged-out users cannot access this page.